src/Enum/EnvironmentEnum.php

```diff
@@ -4,23 +4,24 @@
 
 namespace Answear\OverseasBundle\Enum;
 
-use MabeEnum\Enum;
-use MabeEnum\EnumSerializableTrait;
-
-class EnvironmentEnum extends Enum implements \Serializable
+enum EnvironmentEnum: string implements \JsonSerializable
```

Comment:
Is this interface `JsonSerializable` needed here? From what I see, it will work the same without it.